core/cbor/cbor.go

```diff
@@ -2,20 +2,19 @@ package cbor
 
 import (
 	"bytes"
-	"encoding/json"
 	"fmt"
 	"math/big"
 
 	"github.com/fxamacker/cbor/v2"
+
 	"github.com/smartcontractkit/chainlink/core/store/models"
 )
 
-// ParseDietCBOR attempts to coerce the input byte array into valid CBOR
-// and then coerces it into a JSON object.
+// ParseDietCBOR attempts to coerce the input byte array into valid CBOR.
 // Assumes the input is "diet" CBOR which is like CBOR, except:
 // 1. It is guaranteed to always be a map
 // 2. It may or may not include the opening and closing markers "{}"
-func ParseDietCBOR(b []byte) (models.JSON, error) {
+func ParseDietCBOR(b []byte) (interface{}, error) {
```

Comment:
Would the return type not be `map[string]interface{}`?

Reply:
Fixed